skatepark/block-template-parts/header.html

```diff
@@ -0,0 +1,26 @@
+<!-- wp:group -->
+<div class="wp-block-group">
+<!-- wp:site-logo /-->
+<!-- wp:site-title {"style":{"typography":{"textTransform":"uppercase","fontStyle":"normal","fontWeight":"900"}},"fontSize":"large"} /-->
+<!-- wp:site-tagline {"fontSize":"small"} /--></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"className":"nav-links"} -->
+<div class="wp-block-group nav-links"><!-- wp:navigation {"orientation":"horizontal","itemsJustification":"right","isResponsive":true,"style":{"typography":{"fontStyle":"normal","fontWeight":"900","textTransform":"uppercase"}},"fontSize":"small"} -->
+<!-- wp:navigation-link {"label":"Home","type":"custom","url":"http://skateparkdev.local/","kind":"custom","isTopLevelLink":true} /-->
+
+<!-- wp:navigation-link {"label":"About","type":"page","id":10,"url":"http://skateparkdev.local/about/","kind":"post-type","isTopLevelLink":true} /-->
+
+<!-- wp:navigation-link {"label":"FAQs","type":"page","id":12,"url":"http://skateparkdev.local/faqs/","kind":"post-type","isTopLevelLink":true} /-->
+
+<!-- wp:navigation-link {"label":"Contact","type":"page","id":14,"url":"http://skateparkdev.local/contact/","kind":"post-type","isTopLevelLink":true} /-->
+<!-- /wp:navigation -->
+
+<!-- wp:social-links {"iconColor":"primary","iconColorValue":"#000000","className":"items-justified-right is-style-logos-only"} -->
```

Comment:
we could use the primary value here so that it changes when we change palettes

Reply:
Oooo good idea.